### PR TITLE
Add further advice to nvidia flickering

### PR DESCRIPTION
**Pull request by jones-josh**
While slightly tongue-in-cheek in point 3, the advice is the serious result of several evenings' attempts at improving results on my setup.

### CODE REVIEW

File: pages/Nvidia/_index.md [hunk 1]
```diff
@@ -73,6 +73,11 @@ options nvidia NVreg_RegistryDwords="PowerMizerEnable=0x1; PerfLevelSrc=0x2222;
 
 Reboot your computer and it should be working.
 
+If it does not, try:
+- lowering your monitors' refresh rate, as this can stop the flickering altogether
+- using the [Nouveau driver](https://wiki.archlinux.org/title/Nouveau) as mentioned above
+- using an AMD GPU
```
Comment (vaxerski) [inline pages/Nvidia/_index.md:79]: lol. Funny but doesn't belong here in all seriousness.

Comment (jones-josh) [reply pages/Nvidia/_index.md:79]: Fair, but I do think the refresh rate workaround should be at least mentioned

Comment (vaxerski) [reply pages/Nvidia/_index.md:79]: I am just complaining about point 3

Comment (jones-josh) [reply pages/Nvidia/_index.md]: Removed point 3 and added driver version info as per other thread.

**Review by vaxerski — APPROVED** [review]

**Pull request merged**